### ISSUE TITLE
Trouble adding products to basket

### ISSUE THREAD

**Issue by Marcv89** [tryshoppe/shoppe#330]
I'm having troubles adding products to the basket during the tutorial. If I add a product, nothing happens. No error or whatsoever and nothing in the basket.

This is the code:

**products\show.html.erb**
<%= link_to "Add to basket", product_path(@product.permalink), :method => :post %>

**routes.rb**
post 'product/:permalink' => 'products#buy'

**products_controller.rb**
def buy
    @product = Shoppe::Product.find_by_permalink!(params[:permalink])
    current_order.order_items.add_item(@product, 1)
    redirect_to product_path(@product.permalink), :notice => "Product has been added successfuly!"
  end

**Comment by deanperry** [comment]
You may need to add some stock to that product. Go to the products page and under stock, click the pencil icon and enter in the details.

**Comment by Marcv89** [comment]
Didn't think about that, but unfortunately that's not the problem. No difference between stock or no stock.

**Comment by deanperry** [comment]
And you've run through all the steps on the tutorial page? http://tryshoppe.com/docs/tutorials/getting-started

**Comment by Marcv89** [comment]
I did. Even copy/paste every piece of code till the end of http://tryshoppe.com/docs/tutorials/baskets. Maybe any missing gems?
![gemfile](https://cloud.githubusercontent.com/assets/12737047/12513401/eab580d2-c11d-11e5-9797-61f4e7d4551f.png)

**Comment by deanperry** [comment]
Are you running this on Windows? What if you try Rails 4.2.5?

**Comment by Marcv89** [comment]
Yes Windows. I updated it, but still no difference. And I'm using ruby 2.1.5

**Comment by deanperry** [comment]
I've not tried running Shoppe on Windows so I'm not sure. What if you try the example shoppe on our GitHub account?

**Comment by Marcv89** [comment]
I tried but when I follow the instructions and use `rake db:schema:load shoppe:setup shoppe:seed` it gives me this error:

> rake aborted!
> NoMethodError: undefined method `each' for #<String:0x6494310>
> C:/~/shoppe-example/config/environment.rb:5:in `<top (required)>'
> Tasks: TOP => db:schema:load => environment
> (See full trace by running task with --trace)

I also added 
```
gem 'rake', '~> 10.4', '>= 10.4.2'
gem 'tzinfo-data', platforms: [:mingw, :mswin]
```
to the gemfile to fix other errors.

**Comment by cbryant67** [comment]
What did you do to resolve this?  I'm running into the same issue now.
